index.md

```diff
@@ -1,57 +1,198 @@
 ---
 _appTitle:
 _title: MonoGame
+_overrideDocFx: true
 ---
-<div id="big-image-header" />
-<div id="big-image-slogan">
-	<h1>One framework for creating<br/>powerful cross-platform games</h1>
-</div>
-<br/>
-<div id="big-image-quick-start">
-	<a href="/articles/index.html">Get started</a>
-	<a id="fork-button" href="https://github.com/mono/MonoGame">Fork us</a>
-</div>
-<br/>
-<div id="big-image-title">
-	<a id="built-with-monogame" href=""><img id="big-image-logo" src=""/></a><br/>
-	<a id="built-with-monogame" href="https://twitter.com/search?q=BuiltWithMonoGame">#BuiltWithMonoGame</a>
-</div>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-<br/>
-
-<section class="features-showcase">
-<div class="container-xxl content-center">
-<section>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Managed Code</h4>
-	<p>By leveraging C# and other .NET languages on Microsoft and Mono platforms you can write modern, fast, and reliable game code.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Cross-Platform</h4>
-	<p>MonoGame currently supports Windows, MacOS, Linux, iOS/iPadOS, Android, PlayStation 4, PlayStation 5, Xbox One, and Nintendo Switch.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Open-Source</h4>
-	<p>All the code is available to you ensuring you'll have the ability to make changes when you need to or even port to whole new platforms.</p>
-</div>
-<div class="showcase-feature">
-	<!--<img src="https://www.monogame.net/wp-content/themes/monogame/images/feature.png" alt="feature showcase">-->
-	<h4>Community</h4>
-	<p>With 1000s of shipped games we have built up a vibrant community of developers which use MonoGame for both fun and profit.</p>
-</div>
+<section class="hero" >
+	<div id="hero-background-image" />
+	<div class="hero-content">
+		<p class="slogan">
+			One framework for creating<br/>
+			powerful cross-platform games
+		<div class="hero-button-container">
+			<a type="button" class="btn btn-lrg btn-monogame" href="/articles/getting_started/index.html">
+				Getting Started <i class="bi bi-arrow-right"></i>
+			</a>
+			<a type="button" class="btn btn-lrg btn-monogame" href="https://github.com/MonoGame/MonoGame">
+				Fork Us <i class="bi bi-github"></i>
+			</a>
+		</div>
+	</div>
+	<div class="hero-credits">
+		<a href=""><img id="credits-logo" src=""></a><br/>
+		<a id="credits-built-with" href="https://twitter.com/search?q=BuiltWithMonoGame">#BuiltWithMonoGame</a>

```

Comment:
This #BuiltWithMonoGame looks really hard to read.

That said... since Twitter is dying are hashtags a thing we care about anymore?

Reply:
I adjusted it to bold and have a text shadow to give it more emphasis.   https://github.com/MonoGame/monogame.github.io/pull/19/commits/0190c2fbee945aba8a7a89b881ced4ac4cece7d8

It more noticeable but depending on the background image in that section the colors could still blend like with Stardew Valley and Ty

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/419793a6-f097-48b2-8111-0c2f7c319429)

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/1d2bfca3-0d3e-4c1c-91c9-3214fdb49800)


Another option would be to adjust it to white text.  All links on the page are set to orange color, which is why this picks up that color value, but setting this one specifically to white make sense to make it more visible no matter the background image

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/ed1faf1e-2f03-4e7f-a9d2-cba815140395)

![image](https://github.com/MonoGame/monogame.github.io/assets/103014489/21ca31ae-7d35-4bad-9f7d-075ee8ea61ce)